bin/ci

```diff
@@ -2,4 +2,4 @@
 
 source $(dirname $0)/config
 
-./sbt -Dsbt.log.noformat=true "; clean; stats; test-only -- exclude integration -- console junitxml; publish;  echo-version"
```

Comment:
(that is they shouldn't run, because they can take excessively long - up to an hour sometimes if there is contention or delay)